go/arrow/datatype_fixedwidth.go

```diff
@@ -52,13 +53,25 @@ type (
 	Duration  int64
 )
 
+// func (d Date32) String() string {
```

Comment:
why the commented out code?

Reply:
Ah, that was from an early attempt at trying to simplify the `String()` functions for the scalars, must have forgotten to delete it. I'll get rid of it.